packages/checkbox/src/Checkbox.ts

```diff
@@ -112,7 +112,7 @@ export class Checkbox extends SizedMixin(CheckboxBase) {
                 ${checkmarkIcon[this.size as DefaultElementSize]}
                 ${dashIcon[this.size as DefaultElementSize]}
             </span>
-            <label id="label"><slot></slot></label>
+            <label id="label" for="input"><slot></slot></label>
```

Comment:
This `<input>` already leverages `aria-labelledby="label"`, is one better than the other?

Reply:
`for` makes it so that clicking the label activates focuses and clicks the input, which provides a greater affordance. It's a bit odd to use CSS to make the input the size of the label.

Reply:
Yeah, Spectrum is gonna Spectrum, I guess. But, there is something nice about how the `<input>` prevents double clicks from selecting text without preventing text selection in these controls.

`for` makes sense, and will be a good coverage in case the upstream CSS moves away from that pattern in the future!